src/main/java/lotto/domain/lotto/LottoTicket.java

```diff
@@ -0,0 +1,47 @@
+package lotto.domain.lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class LottoTicket {
+
+    private List<LottoNumbers> lottoTicket;
+
+    public LottoTicket(int quantity) {
+        checkQuantity(quantity);
+        this.lottoTicket = addLottoNumbers(quantity);
+    }
+
+    private void checkQuantity(int quantity) {
+        if (quantity < 1) {
+            throw new IllegalArgumentException("구매 장수가 없습니다.");
+        }
+    }
+
+    private List<LottoNumbers> addLottoNumbers(int quantity) {
+        lottoTicket = new ArrayList<>();
+        while (quantity > 0) {
+            lottoTicket.add(new LottoNumbers());
+            quantity--;
+        }
+        return lottoTicket;
```

Comment:
Stream을 사용하면 더 깔끔하게 바꿀 수 있을 것 같습니다:)
```suggestion
        return IntStream.range(0, quantity)
                .mapToObj(i -> new LottoNumbers())
                .collect(Collectors.toList());
```
(IDE에서 작성한게 아니라서, 오타가 있을 수 있습니다^^; 참고만 해주세요)